game/scripts/npc/abilities/viper_nethertoxin.txt

```diff
@@ -21,7 +21,7 @@
       "01"
       {
         "var_type"          "FIELD_FLOAT"
-        "damage"            "30 45 60 75 100 150"
+        "damage"            "30 45 60 75 150 175"
```

Comment:
Isn't viper already scary good?

Reply:
Nah. Viper is not popular since the nerf.